packages/core/test/__snapshots__/Input.js.snap

```diff
@@ -11,15 +11,15 @@ exports[`Input it renders 1`] = `
   color: inherit;
   background-color: transparent;
   border-radius: 2px;
-  border-width: 0px;
+  border-width: 1px;
   border-style: solid;
+  border-color: #c0cad5;
```

Comment:
this one too has `border-color` twice a few times

Reply:
Addressed in https://github.com/priceline/design-system/pull/683